tools/building.py

```diff
@@ -579,6 +579,14 @@ def closure_compiler(filename, advanced=True, extra_closure_args=None):
   if extra_closure_args:
     user_args += extra_closure_args
 
+  def uses_webgpu_externs(args):
+    for i, arg in enumerate(args):
+      if arg.startswith('--externs=') and 'webgpu-externs.js' in arg:
+        return True
+      if arg == '--externs' and i + 1 < len(args) and 'webgpu-externs.js' in args[i + 1]:
+        return True
+    return False
```

Comment:
Maybe just keep this simple and do `return any('webgpu-externs.js' in a for a in args)`.  

Does it matter if `webgpu-externs-fixes.js` is included even when its not strictly needed?

I assume `webgpu-externs.js` is part of the emdawn project?   If so we should probably file a bug there to get it fixed and leave commend here to remove this code once its fixed.

Reply:
@sbc100 good idea, upstream issue filed: https://issues.chromium.org/issues/472022911